### PR TITLE
magento/magento2#13296: Category name with special characters brakes …

### PR DESCRIPTION
**Pull request by vinayshah**
- Fixed Issue with double quotes in category URL rewrite tree
- Issue regarding test resolved

merge request for 2.2-develop submitted (#13397)
### Description

Escape special character with double quotes or single quotes while displaying tree in the URL rewrite page.

### Fixed Issues (if relevant)

1. magento/magento2#13296: Category name with special characters brakes in url rewrites category tree

### Manual testing scenarios

1. Create a category with quotes (") in its name. eg. Test "Category"
2. Try to create a URL rewrite (Go to Admin>>Marketing>>URL Rewrites>>Add URL Rewrite)
3. Select For Category in the drop-down
4. Then category tree get's loaded below

Review [screenshot](https://drive.google.com/file/d/1NB4XQpuXL7dTnD_mheRcHc4YuQmFM8Xx/view)

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by ishakhsuvarov** [comment]
Hi @vinayshah 
I have removed your assignment from the Pull Requests as it is intended to assign people who are doing actual processing and merging of the PR.
Thanks

**Comment by vinayshah** [comment]
Hi @ishakhsuvarov ,
Okay, no problem. I didn't knew about it.
Thanks.

File: lib/web/underscore.js [hunk 1]
```diff
@@ -1327,7 +1327,8 @@
     '>': '&gt;',
     '"': '&quot;',
     "'": '&#x27;',
-    '`': '&#x60;'
+    '`': '&#x60;',
```
Comment (VladimirZaets) [inline lib/web/underscore.js:1330]: Hi @vinayshah. Underscore.js is a separate library, we can't make changes to it. I think we can add a wrapper to this method inside `mageUtils`.

**Review by VladimirZaets — CHANGES_REQUESTED** [review]

File: lib/web/underscore.js (hunk 1 above)
Comment (VladimirZaets) [inline lib/web/underscore.js:1330]: Hi @vinayshah. Underscore.js is a separate library, we can't make changes to it. I think we can add a wrapper to this method inside `mageUtils`.

**Review by VladimirZaets — CHANGES_REQUESTED** [review]

**Comment by magento-engcom-team** [comment]
Hi @vinayshah. Thank you for your contribution. 
Changes from your Pull Request will be available with the upcoming 2.3.0 release.

**Pull request merged**